schemas/v0.1/sample.yml

```diff
@@ -0,0 +1,48 @@
+# yaml-language-server: $schema=./yaml.schema.json
+
+version: v0.1
+
+workingDir: ai-project/	# default artifacts folder
+
+environmentFolder: ai-project/env/	# .env files location
+
+dataset:        # dataset feature definition
+  content:      # recognizable dataset files
+    - dataset/output/*.json 	# glob pattern
+
+evaluation:     # evaluation feature definition
+  result:       # recognizable evaluation results
+    - evaluation/result/*.json	# glob pattern
+
+
+finetuning: # finetuning feature definition
+  runOn: containerapp # running on Azure Container Apps (ACA)
+  provision:  # provision step definition
+    subscriptionId: ${{AZURE_SUBSCRIPTION_ID}}
+    resourceGroupName: ${{AZURE_RESOURCE_GROUP_NAME}}
+    templates:
+      - path: infra/finetuning.bicep
```

Comment:
Do we need some config to set up the secrets? e.g. `HF_TOKEN`?

Reply:
This can be another command to set secrets to .env file.
Or, may prefer letting users to add ignored .env files by themselves (First stage does not include encryption).